cvat/apps/documentation/user_guide.md

```diff
@@ -1,232 +1,446 @@
+    Use it for interpolation mode. There are several options for using the parameter:
+    - For an interpolation task (video sequence).
+    If you annotate a bounding box on two adjacent segments they will be merged into one bounding box.
+    If overlap equals to zero or annotation is poor on adjacent segments inside a dumped annotation file,
+    you will have several tracks, one for each segment, which corresponds to the object.
+    - For an annotation task (independent images).
+    If an object exists on overlapped segments, the overlap is greater than zero
+    and the annotation is good enough on adjacent segments, it will be automatically merged into one object.
+    If overlap equals to zero or annotation is poor on adjacent segments inside a dumped annotation file,
+    you will have several bounding boxes for the same object.
+    Thus, you annotate an object on the first segment.
+    You annotate the same object on second segment, and if you do it right, you
+    will have one track inside the annotations.
+    If annotations on different segments (on overlapped frames)
+    are very different, you will have two shapes for the same object.
+    This functionality works only for bounding boxes and polygons
+    Polyline, points don't support automatic merge on overlapped segments
+    even the overlap parameter isn't zero and match between corresponding shapes on adjacent segments is perfect.
+
+    **Segment size**. Use this option to divide a huge dataset into a few smaller segments.
+    For example, one job cannot be annotated by several labelers (it isn't supported).
+    Thus using "segment size" you can create several jobs for the same annotation task.
+    It will help you to parallel data annotation process.
+
+    **Image Quality**. Use this option to specify quality of uploaded images.
+    The option helps to load high resolution datasets faster.
+    Use the value from ``1`` (completely compressed images) to ``95`` (almost not compressed images).
+
+    **Select files**. Push this button to select files you want to annotate.
+
+    Push ``Submit`` button and it will be added into the list of annotation tasks.
+    Then, the created task will be displayed on dashboard:
+
+    ![](static/documentation/images/image006.jpg)
 
+1.  The Dashboard contains elements and each of them relates to a separate task. They are sorted in creation order.
+    Each element contains: task name, preview, execution status, buttons, and one or more links.
+    Each button is responsible for a specific function:
+    - ``Dump Annotation`` — download an annotation file from the task (xml format)
+    - ``Upload Annotation`` — uploading an annotation file to the task (xml format)
+    - ``Update Task`` — bring up "Update task" panel. It is used to edit or add labels line
+    - ``Delete Task`` — delete the task
+    - ``Git Repository Sync`` — sync annotation with the repository. Presence depends on task configuration
```

Comment:
Presence depends on task configuration --> It is available only if you specify `dataset repository` when the task has been created.